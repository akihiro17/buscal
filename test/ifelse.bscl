var s1: str = "this is a pen";
var s2: str = "this is an apple";
if s1 == s2 {
  echo("s1 equals to s2");
} else {
  echo("s1 does not equal to s2");
}

var i1: i64 = 120;
var i2: i64 = 120;
var i3: i64 = 1;

if i1 == i2 {
  echo("i1 equals to i2");
}

if i1!=i3 {
  echo("i1 does not equals to i3");
}
