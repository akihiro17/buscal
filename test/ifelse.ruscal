var s1: str = "this is a pen";
var s2: str = "this is an apple";
if s1 == s2 {
  echo("s1 equals to s2");
} else {
  echo("s1 does not equal to s2");
}
