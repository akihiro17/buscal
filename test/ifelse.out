s1 does not equal to s2
i1 equals to i2
i1 does not equals to i3
